src/Dfc.CourseDirectory.WebV2/Features/Courses/DeleteCourseRun.cshtml

```diff
@@ -0,0 +1,87 @@
+    <div class="govuk-grid-column-two-thirds">
+        <form asp-action="DeleteCourseRun" asp-route-courseId="@Model.CourseId" asp-route-courseRunId="@Model.CourseRunId" asp-route-returnUrl="@returnUrl">
+            <h1 class="govuk-heading-xl">Delete course</h1>
+
+            <p class="govuk-body">Deleting this course will remove it from Publish to the course directory and National Careers Service, Find a course.</p>
+
+            <govuk-summary-list>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Course name</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@Model.CourseName</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+                @if (!string.IsNullOrWhiteSpace(Model.YourReference))
+                {
+                    <govuk-summary-list-row>
+                        <govuk-summary-list-row-key>Your reference</govuk-summary-list-row-key>
+                        <govuk-summary-list-row-value>@Model.YourReference</govuk-summary-list-row-value>
+                    </govuk-summary-list-row>
+                }
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Course type</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@Model.DeliveryMode.ToDisplayName()</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Start date</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@(Model.FlexibleStartDate ? "Flexible" : Model.StartDate.Value.ToString("dd/MM/yyyy"))</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+                @if (Model.VenueName != null)
+                {
+                    <govuk-summary-list-row>
+                        <govuk-summary-list-row-key>Course location</govuk-summary-list-row-key>
+                        <govuk-summary-list-row-value>@Model.VenueName</govuk-summary-list-row-value>
+                    </govuk-summary-list-row>
+                }
+            </govuk-summary-list>
+
+            <govuk-checkboxes class="govuk-checkboxes--small" asp-for="Confirm">
+                <govuk-checkboxes-fieldset>
+                    <govuk-checkboxes-fieldset-legend class="govuk-fieldset__legend--m">
+                        <h2 class="govuk-fieldset__heading">
+                            Are you sure?
+                        </h2>
+                    </govuk-checkboxes-fieldset-legend>
+                </govuk-checkboxes-fieldset>
+                <govuk-checkboxes-item value="@(true)">
+                    Yes - I want to delete the course
+                </govuk-checkboxes-item>
+            </govuk-checkboxes>
+
+            <govuk-button type="submit" class="govuk-button--warning" id="pttcd-courses__delete-course__btn">Delete</govuk-button>
```

Comment:
I wonder if this needs visually hidden context for what is being deleted. Probably would benefit from some a11y testing.

Reply:
This lines up with what's in the prototype